### ISSUE TITLE
Improve the readability of the terminal output by increasing the color contrast for the vm startup

### ISSUE THREAD

**Issue by rpkoller** [abiosoft/colima#372]
### Description

Currently when the vm is started there is a "subsection" of terminal output aside the regular output prefixed with `INFO [0000]`. That subsection has a way too low color contrast which is barely readable:  

<img width="1167" alt="Screenshot 2022-07-25 at 17 33 46 (2)" src="https://user-images.githubusercontent.com/1665422/180819053-e7096309-c9de-41ec-bc84-cee8415e5563.png">

Since it is unpredictable what type of terminal profile with what selection of colours will be used I think the easiest fix would be not to use an extra output color complementary to the standard one but use the same output color for everything? 

```
$> colima version
colima version 0.4.4
git commit: 8bb1101a861a8b6d2ef6e16aca97a835f65c4f8f

runtime: docker
arch: aarch64
client: v20.10.17
server: v20.10.11
```

**Comment by jeromeyackley** [comment]
Is there a `--no-color` option for `colima start`?

TL;DR;
I was thinking a `--no-color` option would be a good work-around for this issue.  I've looked through the help, the tests, and the code; and I don't see one.

**Comment by beauwest** [comment]
This is still an issue, and appears to have been reported in #821 as well. I'm personally running a light theme, and the text is nearly invisible due to the theme colors.

<img width="879" height="184" alt="Image" src="https://github.com/user-attachments/assets/07f32b87-e2c2-4738-b6a4-6f071d581ad4" />
